qbraid/runtime/origin/job.py

```
@@ -0,0 +1,140 @@
+            from qbraid.runtime.origin.provider import _get_qcloud_job
+
+            self._backend_job = _get_qcloud_job(self.id, service=self._service)
+        except Exception as exc:
+            raise OriginJobError(f"Unable to retrieve OriginQ job {self.id}") from exc
+        return self._backend_job
+
+    def status(self) -> JobStatus:
+        """Return the current status of the OriginQ job."""
+        try:
+            raw_status = self._get_backend_job().status()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)